core/state_processor.go

```diff
@@ -128,10 +129,14 @@ func (p *StateProcessor) Process(block *types.Block, statedb *state.StateDB, cfg
 				slotnr := rawdb.ReadPreimage(statedb.Database().DiskDB(), stIt.Hash())
 				mkv.addStorageSlot(addr, slotnr, stIt.Slot())
 			}
+			if count == maxMovedCount {
```

Comment:
I think I mentioned something some days ago that we forgot to do; skipping the "first" position.

- If `LastAccHash` is an EOA, then it should be skipped (since `AccountIterator` would repeat again the migration logic)
- If `LastAccHash` isn't an EOA, then we should skip the first result of the `stIt` to not double-migrate the last migrated storage slot.